sources/web/datalab/polymer/components/datalab-files/datalab-files.html

```diff
@@ -188,6 +195,9 @@
             </div>
           </template>
         </div>
+        <paper-button id="toggleDetails" class="navbar-button" on-click="_toggleDetailsPane">
+          <iron-icon icon="visibility"></iron-icon>
```

Comment:
"visibility" doesn't seem quite right to me here. I admit I don't see a better standard material icon, but maybe I missed something, or maybe we can make a custom "preview" icon that is more appropriate.

As an aside, how the heck did the "rowing" icon get into the material icon set, and why is it in the Action section? Maybe it should have been in the Maps section as "directions row", next to "directions walk" and "directions run".